### PR TITLE
[WIP] Fix input type on android using SDL2 as window provider

### PR DESCRIPTION
**Pull request by Franccisco**
This pull request allows the use of `input_type` and `keyboard_suggestions` properties to modify the keyboard displayed on Android. In summary, it uses the `changeKeyboard` function in PythonActivity.java (see kivy/python-for-android#2057). Additionally, `keyboard_suggestions` was moved from kivy/uix/textinput.py to kivy/uix/behaviors/focus.py, so it's more accesible to send to the  `request_keyboard` method.

This is ready to review. Note that you may encounter some bugs with these changes:
- Inserting any emoji will crash the app in older Android versions, and a tofu will appear in latest versions.
- Keyboard suggestions have a weird behavior. You won't see anything you type until you select the suggestion.

The details are in (kivy/python-for-android#2015).  Unless those bugs are solved, this PR shouldn't be merged.

### CODE REVIEW

File: kivy/uix/behaviors/focus.py [hunk 1]
```diff
@@ -237,6 +237,16 @@ def _get_keyboard(self):
         2.0.0.
     '''
 
+    keyboard_suggestions = BooleanProperty(True)
+    '''If True provides auto suggestions on top of keyboard.
+    This will only work if :attr:`input_type` is set to `text`, `url`, `mail` or
+    `address`.
+
+    .. versionadded:: 1.8.0
```
Comment (matham) [inline kivy/uix/behaviors/focus.py]: 2.0.0

Comment (Franccisco) [reply kivy/uix/behaviors/focus.py]: Modified. Thank you!

**Review by mannutheman3 — COMMENTED** [review]
This give TypeError Request_Keyboard() takes from 3 to 4 positional arguments but 5 were given.

**Comment by matham** [comment]
Closing for https://github.com/kivy/kivy/pull/7231.

**Pull request closed without merging**